ui/src/views/ProjectsExplorer.vue

```diff
@@ -431,7 +431,7 @@ export default defineComponent({
           this.successMessage = `Successfully deleted file [${file}] from directory [${folder}] of project: [${this.projectId}]`;
         })
         .catch((error) => {
-          this.errorMessage = error;
+          this.errorMessage = `${error}`;
```

Comment:
unneeded change?

Reply:
I just tried to "fix" this, but this is in fact correct. There was an issue with typing. `error` is of type `Error`, but this error is of type `string`

Reply:
Isn't error a reserved word?